src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -579,14 +579,6 @@ public virtual Result VisitResult(Result node)
                     }
                 }
 
-                if (node.Graphs != null)
```

Comment:
>if [](start = 16, length = 2)

I don't know why this generated code used to visit the elements of the array, but it is not visiting the elements of the dictionary. Before I merge, I'll either fix this, or, if it's not a regression, file an issue for it.

Reply:
I found it. This is a bug in the code generation. It does not visit dictionaries at all. (It doesn't visit `resources.rules` either, for example. I will file a bug in JSchema and address it next.

---
In reply to: [217210637](https://github.com/Microsoft/sarif-sdk/pull/1085#discussion_r217210637) [](ancestors = 217210637)

Reply:
Correction: It does visit _some_ dictionaries, in particular `run.files`. So that is a clue. I filed [https://github.com/Microsoft/jschema/issues/37](https://github.com/Microsoft/jschema/issues/37), "RewritngVisitor does not visit some dictionaries." @michaelcfanning FYI.

---
In reply to: [217465104](https://github.com/Microsoft/sarif-sdk/pull/1085#discussion_r217465104) [](ancestors = 217465104,217210637)

Reply:
Update: The root cause is CodeGenHints errors. The code generator is fine. Will address in separate PR.

---
In reply to: [217469896](https://github.com/Microsoft/sarif-sdk/pull/1085#discussion_r217469896) [](ancestors = 217469896,217465104,217210637)